### ISSUE TITLE
[Feature] Recurrent events: Choice cast

### ISSUE THREAD

**Issue by Germano123** [gameguild-gg/gameguild#5]
In our goals, we want to have an event every week. To do this we give the community several options to choose an event of collective interest. This feature will serve as a guide so that we can better control the event that will happen weekly.

**Comment by tolstenko** [comment]
@Germano123 I guess this is related to the issue #47 and it can be done via web3. If so, we will postpone until v3.

**Comment by Germano123** [comment]
I don't think so. I see this feature as something that needs week flush. Think in every week workshop with different proposals and people that want some space to talk about something. I think this should be handled with simple back-end logic. Gonna move into this as soon as I finish course enrollment #52

**Comment by tolstenko** [comment]
@Germano123 are you still working on this?

**Comment by Germano123** [comment]
Yes. And I'm still in it. Last week I reviewed the documentation in [requirements.md](https://github.com/gameguild-gg/website/blob/main/Requirements.md) and now I'm putting together the action flow for this functionality.

**Comment by tolstenko** [comment]
@Germano123, How is this feature going? It is around 1y and a half in your hands. Can you give me a timeline for this?

**Comment by Germano123** [comment]
I'm gonna put myself a deadline into this. Next week I'm gonna deliver this. Otherwise I'm gonna deassign myself from this task.
